synapse/types.py

```diff
@@ -157,8 +156,8 @@ def deserialize(store, input):
 def create_requester(
     user_id: Union[str, "UserID"],
     access_token_id: Optional[int] = None,
-    is_guest: Optional[bool] = False,
-    shadow_banned: Optional[bool] = False,
+    is_guest: bool = False,
+    shadow_banned: bool = False,
```

Comment:
and these

Reply:
Can we remove the types in the docstring while we're cleaning this up?

Reply:
done